MasterForm.Designer.cs

```diff
@@ -129,7 +129,7 @@ partial class MasterForm {
             // 
             this.megaCaution.Anchor = ((System.Windows.Forms.AnchorStyles)((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Right)));
             this.megaCaution.Image = ((System.Drawing.Image)(resources.GetObject("megaCaution.Image")));
-            this.megaCaution.Location = new System.Drawing.Point(220, 4);
+            this.megaCaution.Location = new System.Drawing.Point(217, 4);

```

Comment:
Be careful when moving UI elements around. The majority of things in this app are located with coordinates, so moving the position of elements could cause the program to stack trace under certain conditions. I was worried about this here with the megaCaution icon, but it looks like I don't use coords to access it, so it's fine. Just keep this in mind for the future.

Reply:
Any shifts in positioning are artifacts of me migrating the elements to tab control (cut and paste in the designer) lets not push this PR until all of them are reverted to avoid messing up your positioning code.

Reply:
See the below comment.